packages/components/src/icon-button/index.js

```diff
@@ -42,7 +43,7 @@ class IconButton extends Component {
 
 		let element = (
 			<Button aria-label={ label } { ...additionalProps } className={ classes }>
-				{ isString( icon ) ? <Dashicon icon={ icon } /> : icon }
+				{ isString( icon ) ? <Dashicon icon={ icon } ariaPressed={ariaPressed}/> : icon }
```

Comment:
`aria-pressed` is specific to the web. Instead of trying to hijack that, I'd prefer if we rename it to something more neutral (`pressed`?), then make sure each platform does what it needs to render correctly.

Reply:
I have that on a mind, but it seems that we are using aria-pressed in button/index.native.js so I wanted to be consistent with other parts of the code.